ts/parts/Chart.ts

```diff
@@ -2888,3 +2888,45 @@ extend(Chart.prototype, /** @lends Highcharts.Chart.prototype */ {
     }
 
 }); // end Chart
+
+// Extend the Axis prototype to calculate start min/max values
+// (including min/maxPadding). This is related to using vertical panning
+// (#11315).
+addEvent(Axis, 'afterSetScale', function (
+    this: Highcharts.Axis
+): void {
+    var axis = this,
+        chartOptions = axis.chart.options.chart,
+        panning = chartOptions &&
+            chartOptions.panning;
+
+    if (
+        panning &&
+        (
+            panning.type &&
+            panning.type.match('y')
+        ) &&
+        !axis.isXAxis &&
+        !defined(axis.panningState) &&
+        axis.series.length
+    ) {
+        let min: (number|undefined),
+            max: (number|undefined);
+
+        axis.series.forEach(function (series): void {
+            if (!(isNumber(min) && isNumber(max))) {
+                min = Number.MAX_VALUE;
+                max = Number.MIN_VALUE;
+            }
+            min = Math.min(H.arrayMin(series.yData as any), min) -
```

Comment:
Get rid of the `any` casts. The whole block can be wrapped in a type check for `series.yData`